lib/views/home/home_view.dart

```diff
@@ -35,10 +36,21 @@ class HomeViewState extends ConsumerState<HomeView> {
         },
       );
 
+  Widget get _body => Consumer(
+        builder: (_, ref, __) {
+          final surveyList = ref.watch(surveyListStream).value ?? [];
+          final profileUrl = ref.watch(profileImageUrlStream).value ?? '';
+          return surveyList.isNotEmpty && profileUrl.isNotEmpty
```

Comment:
@nmint8m It's not good to check both `surveyList` and `profileUrl` to display `HomeHeader`. I suggest always showing `HomeHeader` with the default avatar. Besides, we only display `HomeSkeletonLoading` on the loading state, not on checking data.
In case we only need to display the skeleton loading, just show it without any check.
What do you think?

Reply:
I won't check the `profileUrl`. However, I need to check `surveyList` to display the skeleton loading when the list is empty.